docs/pages.rst

```diff
@@ -64,3 +64,13 @@ e.g. ``pages/index.md``::
 
     We invite you to [join us](/attend/) at [our venue](/venue/)
     on the 31st of December for a day of fun conferencing.
+
+The front matter can contain a couple of flags:
+
+``published``
+    If set to ``false``, the page will not be loaded.
```

Comment:
Oh, I see. Perhaps we should explain that it is not loaded by the `load_pages` management command?